torch/fx/experimental/symbolic_shapes.py

```diff
@@ -428,8 +428,16 @@ def wrapper(self, *args, **kwargs):
 
 # This stub exists so we can easily add metadata to sympy symbols
 class Symbol(sympy.Symbol):
-    __slots__: List[str] = []
+    __slots__: List[str] = ['snames', 'shape_env']
+    snames: List[str]
```

Comment:
In the end, it was pretty useful for debugging purposes to see what snames produced a symbol!